docs/running.md

```diff
@@ -9,14 +9,224 @@ a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet
 
 ## Kubernetes
 
-Deploy the resources in `src/templates/`, this sets up the correct permissions on the cluster (`rbac-config.yaml`) and deploys the warnet RPC server as a service + statefulset.
+Deploy the resources in `src/templates/`, this sets up a Kubernetes cluster, apply correct permissions on the cluster (`rbac-config.yaml`), and deploys the warnet RPC server as a service + statefulset.
 
-This can be done with from inside the `src/templates/` directory by running:
+### Using Justfile
```

Comment:
Can we promote this to a 2-hash header? It seems like the next important step after getting minikube started.